testool/src/statetest/executor.rs

```diff
@@ -78,9 +78,13 @@ fn check_post(
 
         if let Some(expected_code) = &expected.code {
             let actual_code = if actual.code_hash.is_zero() {
-                std::borrow::Cow::Owned(Vec::new())
+                vec![]
             } else {
-                std::borrow::Cow::Borrowed(&builder.code_db.0[&actual.code_hash])
```

Comment:
I don't get why we currently need `std::borrow::Cow::Owned` and `std::borrow::Cow::Borrowed`

Reply:
I discussed this with @ed255. We concluded that the Cow was there to prevent cloning twice.

- Cow is used to keep `actual_code` a reference to the byte vector so that we don't copy bytecodes from codeDB here.
- The line `found: Bytes::from(actual_code.to_vec()),` below clones bytes with the `to_vec` method.

I resolved with another method that only clones once.